common/src/main/org/jetbrains/lincheck/descriptors/CodeLocations.kt

```diff
@@ -12,14 +12,17 @@ package org.jetbrains.lincheck.descriptors
 
 import org.jetbrains.lincheck.trace.TraceContext
 import java.util.Objects
+import kotlin.String
+import kotlin.collections.List
 
 class CodeLocation(
     val stackTraceElement: StackTraceElement,
     val accessPath: AccessPath? = null,
 
     // TODO: this only makes sense for method call code locations,
     //   consider introducing proper type hierarchy for code locations
-    val argumentNames: List<AccessPath?>? = null
+    val argumentNames: List<AccessPath?>? = null,
+    val activeLocals: List<String>? = null
```

Comment:
Would it be difficult to pass variable ids instead (and decode them to `VariableDescriptor` on plugin side)?
I think it would be more consistent approach in the long run. 

If it is too complicated for now, please create a YT ticket.